src/Catty/PocketPaint/Tools/DrawTool.m

```diff
@@ -40,18 +40,21 @@ - (id) initWithDrawViewCanvas:(PaintViewController *)canvas
 
 - (void)draw:(UIGestureRecognizer *)recognizer
 {
-
+  CGPoint point;
   if (recognizer.state == UIGestureRecognizerStateBegan){
       if (self.canvas.isEraser) {
           self.canvas.saveView.hidden = YES;
           self.canvas.drawView.image = self.canvas.saveView.image;
           self.canvas.drawView.backgroundColor = self.canvas.saveView.backgroundColor;
       }
-      CGPoint point = [recognizer locationOfTouch:0 inView:self.canvas.drawView];
+      if (recognizer.numberOfTouches) {
+          point = [recognizer locationOfTouch:0 inView:self.canvas.drawView];
+      }
```

Comment:
Unfortunately we do not have a Linter for Objective C code yet. Among others because we want to convert all our code into Swift in the long run. However, please reformat those lines as follows:

```
if () {
...
} else {
...
}
```